explorer/ast/declaration.cpp

```diff
@@ -15,104 +15,112 @@ using llvm::cast;
 Declaration::~Declaration() = default;
 
 void Declaration::Print(llvm::raw_ostream& out) const {
+  PrintDepth(-1, 0, out);
+}
+void Declaration::PrintDepth(int depth, int indent_num_spaces,
+                             llvm::raw_ostream& out) const {
+  if (depth == 0) {
+    out.indent(indent_num_spaces) << " ... ";
```

Comment:
Can this indent once at the start of the function, for all declarations, avoiding the call at the start of each declaration?

Reply:
Sure, I'll do that